lib/compress.js

```diff
@@ -706,9 +706,9 @@ merge(Compressor.prototype, {
         return x;
     };
 
-    function is_iife_call(node) {
+    function is_iife_call(node, traditional) {
         if (node instanceof AST_Call && !(node instanceof AST_New)) {
-            return is_func_expr(node.expression) || is_iife_call(node.expression);
+            return is_func_expr(node.expression, traditional) || is_iife_call(node.expression);
```

Comment:
It's not always true - it's called in another context where it will be false (undefined actually).

https://github.com/kzc/UglifyJS2/blob/79a5683fff9ef9fa679ba48c78c3d28d5b8ad9ad/lib/compress.js#L2633